packages/modules/src/modules/banners/investigationsMoment/components/InvestigationsMomentBannerPolygon.tsx

```diff
@@ -0,0 +1,123 @@
+import { css, SerializedStyles } from '@emotion/react';
+import { Breakpoint, from, until } from '@guardian/src-foundations/mq';
+import React from 'react';
+
+export type Shadow = 'shadowTopRight' | 'shadowBottomLeft';
+
+export type SvgPolygonProps = {
+    viewBox: string;
+    points: string;
+    shadow: Shadow;
+    hideBelowBreakpoint?: Breakpoint;
+    hideAboveBreakpoint?: Breakpoint;
+};
+
+const styles = {
+    shadowTopRight: (hideBelowBreakpoint?: Breakpoint, hideAboveBreakpoint?: Breakpoint) => css`
+        position: absolute;
+        pointer-events: none;
+        display: flex;
+        justify-content: flex-end;
+        top: 0;
+        right: 0;
+        width: 150px;
+        height: 80px;
+
+        ${from.mobileMedium} {
+            width: 200px;
+        }
+
+        ${from.mobileLandscape} {
+            width: 300px;
+        }
+
+        ${from.phablet} {
+            width: 475px;
+        }
+
+        ${from.tablet} {
+            bottom: 0;
+            width: auto;
+            height: auto;
+        }
+
+        svg {
+            display: block;
+            height: 100%;
+
+            ${from.tablet} {
+                height: 90%;
+            }
+
+            ${from.desktop} {
+                height: 95%;
+            }
+
+            ${from.leftCol} {
+                height: 90%;
+            }
+        }
+
+        // The following styles will conditionally hide the svg polygon
+        ${hideBelowBreakpoint && until[hideBelowBreakpoint]} {
+            display: none;
+        }
+        ${hideAboveBreakpoint && from[hideAboveBreakpoint]} {
+            display: none;
+        }
+    `,
+    shadowBottomLeft: (hideBelowBreakpoint?: Breakpoint, hideAboveBreakpoint?: Breakpoint) => css`
+        position: absolute;
+        pointer-events: none;
+        bottom: 0;
+        left: 0;
+        right: 20px;
+
+        svg {
+            display: block;
+        }
+
+        ${from.wide} {
+            height: 125px;
+            width: 1250px;
+            right: auto;D
+        }
```

Comment:
Good point, have removed both from InvestigationsMomentBanner.tsx. Disadvantage of the (neat) emotions 'styles' object approach where ESlint does not auto-flag unused css. George has mentioned previously perhaps a better approach is to define each emotion css style separately to avoid this. I usually do this on new components.